engine/src/CalciteInterpreter.cpp

```diff
@@ -910,7 +909,7 @@ std::unique_ptr<ral::frame::BlazingTable> evaluate_split_query(std::vector<ral::
 //			}
 //		}
 
-//		ral::communication::network::Server::getInstance().deregisterContext(queryContext.getContextToken());
+//		ral::communication::network::experimental::Server::getInstance().deregisterContext(queryContext.getContextToken());
```

Comment:
do we no longer need to deregister?

Reply:
Don't know I think this was in the legacy code maybe @williamBlazing can answer this

Reply:
we do want to deregister context